.travis.yml

```diff
@@ -18,6 +18,18 @@ jdk:
 before_script:
   - bundle exec rake db:create
 
+before_deploy:
+  - mkdir build; zip -r build/hybox.zip .
```

Comment:
It looks like this creates too long of a log. Can we pass the `-q` (quiet) flag to `zip`?